arm-customer-insights/2017-01-01/swagger/customer-insights.json

```diff
@@ -4765,17 +4773,17 @@
           "type": "string",
           "description": "The primary participant property name for an interaction ,This is used to logically represent the agent of the interaction, Specify the participant name here from ParticipantName."
         },
-        "dataSources": {
```

Comment:
hi sorry looks like it's released before. However this part was incomplete so I'm sure no one was using it. Anyway the version 2017-01-01 should look like this swagger now